tests/manual-test-cases/Group18-VIC-UI/vicui-common.robot

```diff
@@ -105,29 +107,18 @@ Set Script Filename
     ${SCRIPT_FILENAME}=  Join Path  ${UI_INSTALLER_PATH}  ${SCRIPT_FILENAME}
     Set Suite Variable  \$${suite_varname}  ${SCRIPT_FILENAME}
 
-Set Vcenter Ip
-    # Populate VCENTER_IP with ${TEST_VC_IP}
-    Remove File  ${UI_INSTALLER_PATH}/configs
-    ${results}=  Replace String Using Regexp  ${configs}  VCENTER_IP=.*  VCENTER_IP=\"${TEST_VC_IP}\"
-    ${results}=  Run Keyword If  ${TEST_VC_VERSION} == '5.5'  Replace String Using Regexp  ${results}  IS_VCENTER_5_5=.*  IS_VCENTER_5_5=1  ELSE  Set Variable  ${results}
-    Create File  ${UI_INSTALLER_PATH}/configs  ${results}
-    ${check}=  OperatingSystem.Get File  ${UI_INSTALLER_PATH}/configs
-    Should Contain  ${check}  ${TEST_VC_IP}
-
 Unset Vcenter Ip
     # Revert the configs file back to what it was
     #Remove File  ${UI_INSTALLER_PATH}/configs
-    ${results}=  Replace String Using Regexp  ${configs}  VCENTER_IP=.*  VCENTER_IP=\"\"
-    ${results}=  Replace String Using Regexp  ${results}  IS_VCENTER_5_5=.*  IS_VCENTER_5_5=0
-    #Generate Config  ${UI_INSTALLER_PATH}/configs  '${results}'
-    Run  echo '${results}' > ${UI_INSTALLER_PATH}/configs
+    ${results}=  Replace String Using Regexp  ${configs}  VIC_UI_HOST_URL=.*  VIC_UI_HOST_URL=\"\"
+    ${results}=  Replace String Using Regexp  ${results}  VIC_UI_HOST_THUMBPRINT=.*  VIC_UI_HOST_THUMBPRINT=\"\"
+    Create File  ${UI_INSTALLER_PATH}/configs  ${results}
     Should Exist  ${UI_INSTALLER_PATH}/configs
 
 Force Remove Vicui Plugin
-    Uninstall Vicui  ${TEST_VC_USERNAME}  ${TEST_VC_PASSWORD}
-    ${output}=  OperatingSystem.GetFile  uninstall.log
-    Should Match Regexp  ${output}  (unregistration was successful|failed to find target plugin)
-    Remove File  uninstall.log
+    # Uninstall Vicui  ${TEST_VC_IP}  ${TEST_VC_USERNAME}  ${TEST_VC_PASSWORD}  ${TRUE}
+    ${rc}  ${output}=  Run And Return Rc And Output  ../../../ui-nightly-run-bin/vic-ui-linux remove --thumbprint %{VC_FINGERPRINT} --target ${TEST_VC_IP} --user ${TEST_VC_USERNAME} --password ${TEST_VC_PASSWORD} --key com.vmware.vic.ui
+    ${rc}  ${output}=  Run And Return Rc And Output  ../../../ui-nightly-run-bin/vic-ui-linux remove --thumbprint %{VC_FINGERPRINT} --target ${TEST_VC_IP} --user ${TEST_VC_USERNAME} --password ${TEST_VC_PASSWORD} --key com.vmware.vic
```

Comment:
This file appears to be what I was commenting about above.  This should be pulled into the resources folder as UI-Util.robot (or some such)